pkg/ingester/ingester.go

```diff
@@ -84,6 +83,7 @@ type Config struct {
 
 	ChunkFilterer storage.RequestChunkFilterer `yaml:"-"`
 	LabelFilterer LabelValueFilterer           `yaml:"-"`
+	Wrapper       Wrapper                      `yaml:"-"`
```

Comment:
Should we add a doc string explaining the idea behind the wrapper?

Reply:
Good idea. A comment was added.